2025-09-04 00:03:26,902 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 00:06:23,956 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:06:39,113 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:06:42,808 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:10:36,831 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 00:16:47,211 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:21:20,870 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:33:10,296 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:37:06,903 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:40:17,369 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:45:19,686 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:46:10,616 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:46:24,806 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:48:48,784 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:49:21,815 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:52:56,896 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:53:00,826 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:53:14,074 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 00:54:23,574 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:57:34,992 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 00:57:43,183 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 00:58:19,247 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 00:58:19,264 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 00:58:19,264 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.017s)
2025-09-04 00:58:19,264 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.017s
2025-09-04 00:58:19,424 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 00:58:29,643 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$40000.0
2025-09-04 00:58:29,657 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$40000.0 (-0.014s)
2025-09-04 00:58:29,657 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 632, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 1, 'subtotal': 32400.0, 'iva': 7600.0, 'total_sell': 40000.0}}], empleado=francisco, total=$40000.0, tipo_pago=transferencia
2025-09-04 00:58:29,658 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.015s
2025-09-04 00:58:38,411 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 00:58:42,284 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 453, in _handle_sent_form
    "nombre": client.get("name"),
              ~~~~~~~~~~^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:00:12,133 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:00:16,500 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 453, in _handle_sent_form
    "nombre": client.get("name"),
              ~~~~~~~~~~^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:02:12,712 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:02:46,818 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:03:44,938 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:04:06,564 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:04:10,384 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 453, in _handle_sent_form
    "nombre": client.name,
              ^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'name'
2025-09-04 01:04:59,584 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:05:03,282 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 453, in _handle_sent_form
    "nombre": client.get("name"),
              ~~~~~~~~~~^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:05:24,599 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:05:27,731 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 453, in _handle_sent_form
    "nombre": client.get("name", 0),
              ~~~~~~~~~~^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:09:08,985 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:09:17,472 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 449, in _handle_sent_form
    client = GetDataClientForBill.get_data_client(client_email_to_send)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 18, in get_data_client
    "name": client_info.get("name"),
            ^^^^^^^^^^^^^^^
AttributeError: 'Clients' object has no attribute 'get'
2025-09-04 01:13:23,170 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:15:00,263 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:15:04,332 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 449, in _handle_sent_form
    client = GetDataClientForBill.get_data_client(client_email_to_send)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 19, in get_data_client
    "name": client_info.get("name"),
            ^^^^^^^^^^^^^^^
AttributeError: 'Clients' object has no attribute 'get'
2025-09-04 01:16:13,529 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:16:16,183 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 449, in _handle_sent_form
    client = GetDataClientForBill.get_data_client(client_email_to_send)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 19, in get_data_client
    "name": client_info.values("name"),
            ^^^^^^^^^^^^^^^^^^
AttributeError: 'Clients' object has no attribute 'values'
2025-09-04 01:16:51,914 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:16:55,029 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 449, in _handle_sent_form
    client = GetDataClientForBill.get_data_client(client_email_to_send)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 20, in get_data_client
    "direcction": client_info.direcction,
                  ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Clients' object has no attribute 'direcction'. Did you mean: 'direction'?
2025-09-04 01:17:01,551 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:17:04,253 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 458, in _handle_sent_form
    "cantidad": sell_product.get("quantity", {}),
                ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:18:18,329 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:18:23,325 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:18:25,617 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 458, in _handle_sent_form
    "cantidad": sell_product.get("quantity", {}),
                ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 01:19:34,230 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:19:37,313 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2237, in add_fields
    join_info = self.setup_joins(
        name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m
    )
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1900, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
                                       ~~~~~~~~~~~~~~~~~~^
        names[:pivot],
        ^^^^^^^^^^^^^^
    ...<2 lines>...
        fail_on_missing=True,
        ^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1805, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'price' into field. Choices are: idproduct, idproduct_id, idsell, idsell_id, idsell_product, priceunitaty, quantity

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 459, in _handle_sent_form
    "precio": sell_product.values("price"),
              ~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1368, in values
    clone = self._values(*fields, **expressions)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1363, in _values
    clone.query.set_values(fields)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2563, in set_values
    self.add_fields(field_names, True)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2265, in add_fields
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'price' into field. Choices are: idproduct, idproduct_id, idsell, idsell_id, idsell_product, priceunitaty, quantity
2025-09-04 01:20:16,056 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:20:18,200 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 84, in create_bill
    total_item = item["cantidad"] * item["precio"]
                 ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for *: 'QuerySet' and 'QuerySet'
2025-09-04 01:23:16,520 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:23:20,478 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 84, in create_bill
    total_item = item.values("quantity") * item.values("priceunitaty")
                 ~~~~~~~~~~~^^^^^^^^^^^^
TypeError: dict.values() takes no arguments (1 given)
2025-09-04 01:23:47,940 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:23:51,582 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 84, in create_bill
    total_item = item.values("quantity") * item.values("priceunitaty")
                 ^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'values'
2025-09-04 01:24:53,400 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:25:52,596 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:25:57,135 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 454, in _handle_sent_form
    "direccion": client[0]["direction"],
                 ~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'direction'
2025-09-04 01:26:29,895 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:26:33,205 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 458, in _handle_sent_form
    "cantidad": float(sell_product.values("quantity")),
                ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: float() argument must be a string or a real number, not 'QuerySet'
2025-09-04 01:28:08,402 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:28:12,325 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 458, in _handle_sent_form
    "cantidad": float(str(sell_product.values("quantity"))),
                ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: could not convert string to float: "<QuerySet [{'quantity': 1}]>"
2025-09-04 01:34:42,429 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:34:47,501 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 89, in create_bill
    item["descripcion"],
    ~~~~^^^^^^^^^^^^^^^
KeyError: 'descripcion'
2025-09-04 01:35:39,693 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:35:44,019 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 95, in create_bill
    iva = subtotal * 0.16  # Ejemplo con 16% de IVA
          ~~~~~~~~~^~~~~~
TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'
2025-09-04 01:36:11,885 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:36:15,306 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 96, in create_bill
    total_final = subtotal + iva
                  ~~~~~~~~~^~~~~
TypeError: unsupported operand type(s) for +: 'decimal.Decimal' and 'float'
2025-09-04 01:36:20,732 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 96, in create_bill
    total_final = subtotal + iva
                  ~~~~~~~~~^~~~~
TypeError: unsupported operand type(s) for +: 'decimal.Decimal' and 'float'
2025-09-04 01:37:00,973 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:37:04,067 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 96, in create_bill
    iva = subtotal * IVA_RATE  # Ejemplo con 16% de IVA
          ~~~~~~~~~^~~~~~~~~~
TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'
2025-09-04 01:37:20,076 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:37:22,038 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 96, in create_bill
    iva = subtotal * float(IVA_RATE)  # Ejemplo con 16% de IVA
          ~~~~~~~~~^~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for *: 'decimal.Decimal' and 'float'
2025-09-04 01:37:36,659 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:37:39,923 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 327, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 470, in _handle_sent_form
    email_message = create_bill("factura_psys.pdf", datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 97, in create_bill
    total_final = subtotal + iva
                  ~~~~~~~~~^~~~~
TypeError: unsupported operand type(s) for +: 'decimal.Decimal' and 'float'
2025-09-04 01:38:05,417 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:50:46,435 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:55:45,870 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:56:52,151 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:59:06,911 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:59:16,966 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 01:59:51,799 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 01:59:58,993 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 01:59:59,008 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 01:59:59,008 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.015s)
2025-09-04 01:59:59,008 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 02:00:06,300 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$40000.0
2025-09-04 02:00:06,314 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$40000.0 (-0.014s)
2025-09-04 02:00:06,316 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 633, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 1, 'subtotal': 32400.0, 'iva': 7600.0, 'total_sell': 40000.0}}], empleado=francisco, total=$40000.0, tipo_pago=transferencia
2025-09-04 02:00:06,316 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.016s
2025-09-04 02:00:16,984 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 331, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 475, in _handle_sent_form
    pdf_buffer = create_bill_in_memory(datos_factura)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py", line 90, in create_bill_in_memory
    item["descripcion"],  # Nota: el original no incluía esto
    ~~~~^^^^^^^^^^^^^^^
KeyError: 'descripcion'
2025-09-04 02:00:30,827 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 02:06:00,455 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:07:20,662 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 02:07:34,019 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/tasks.py changed, reloading.
2025-09-04 02:09:43,676 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/tasks.py changed, reloading.
2025-09-04 02:10:57,307 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/tasks.py changed, reloading.
2025-09-04 02:15:31,532 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:15:35,235 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 02:15:39,670 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 02:15:39,682 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 02:15:39,682 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.012s)
2025-09-04 02:15:39,683 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.013s
2025-09-04 02:15:49,827 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$40000.0
2025-09-04 02:15:49,854 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$40000.0 (-0.027s)
2025-09-04 02:15:49,855 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 634, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 1, 'subtotal': 32400.0, 'iva': 7600.0, 'total_sell': 40000.0}}], empleado=francisco, total=$40000.0, tipo_pago=Efectivo
2025-09-04 02:15:49,855 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.028s
2025-09-04 02:17:15,356 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/tasks.py changed, reloading.
2025-09-04 02:17:21,138 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 02:17:38,390 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 02:17:38,404 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 02:17:38,404 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.014s)
2025-09-04 02:17:38,404 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.014s
2025-09-04 02:17:38,590 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 02:17:45,586 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$40000.0
2025-09-04 02:17:45,601 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$40000.0 (-0.015s)
2025-09-04 02:17:45,602 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 635, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 1, 'subtotal': 32400.0, 'iva': 7600.0, 'total_sell': 40000.0}}], empleado=francisco, total=$40000.0, tipo_pago=Efectivo
2025-09-04 02:17:45,602 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.016s
2025-09-04 02:17:50,629 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 02:20:19,360 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 02:20:28,799 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:23:04,432 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/constants.py changed, reloading.
2025-09-04 02:24:08,641 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:25:16,126 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/constants.py changed, reloading.
2025-09-04 02:26:51,412 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:27:05,568 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:29:17,237 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:31:21,813 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:32:06,098 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:32:42,202 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:32:58,318 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:34:21,549 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 02:34:30,725 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:30,734 - psysmysql.stock - INFO - Resumen de stock generado: 11944 productos
2025-09-04 02:34:30,734 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.009s)
2025-09-04 02:34:30,734 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.009s
2025-09-04 02:34:30,734 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:30,737 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:30,738 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.004s)
2025-09-04 02:34:37,202 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:37,209 - psysmysql.stock - INFO - Resumen de stock generado: 11944 productos
2025-09-04 02:34:37,209 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.006s)
2025-09-04 02:34:37,209 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.007s
2025-09-04 02:34:37,209 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:37,212 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:37,212 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.003s)
2025-09-04 02:34:37,819 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:37,827 - psysmysql.stock - INFO - Resumen de stock generado: 11944 productos
2025-09-04 02:34:37,827 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.009s)
2025-09-04 02:34:37,827 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.009s
2025-09-04 02:34:37,828 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:37,834 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:37,835 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.007s)
2025-09-04 02:34:38,441 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:38,448 - psysmysql.stock - INFO - Resumen de stock generado: 11944 productos
2025-09-04 02:34:38,448 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.007s)
2025-09-04 02:34:38,448 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.007s
2025-09-04 02:34:38,448 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:38,452 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:38,452 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.003s)
2025-09-04 02:34:38,987 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:38,995 - psysmysql.stock - INFO - Resumen de stock generado: 11944 productos
2025-09-04 02:34:38,995 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.008s)
2025-09-04 02:34:38,995 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.008s
2025-09-04 02:34:38,995 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:38,998 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:38,999 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.003s)
2025-09-04 02:34:46,548 - psysmysql.stock - INFO - Iniciando: Buscando item en stock: 22
2025-09-04 02:34:46,549 - psysmysql.stock - INFO - Completado: Buscando item en stock: 22 (-0.001s)
2025-09-04 02:34:46,549 - psysmysql.performance - INFO - psysmysql.services.stock_service.search_item ejecutado en 0.001s
2025-09-04 02:34:46,550 - psysmysql.stock - INFO - Iniciando: Actualizando stock producto 22: add 1
2025-09-04 02:34:46,565 - psysmysql.stock - INFO - Stock actualizado para producto Televisor: 1000
2025-09-04 02:34:46,565 - psysmysql.stock - INFO - Completado: Actualizando stock producto 22: add 1 (-0.015s)
2025-09-04 02:34:46,566 - psysmysql.performance - INFO - psysmysql.services.stock_service.create_or_update_stock ejecutado en 0.015s
2025-09-04 02:34:46,577 - psysmysql.stock - INFO - Iniciando: Generando resumen de stock
2025-09-04 02:34:46,583 - psysmysql.stock - INFO - Resumen de stock generado: 11945 productos
2025-09-04 02:34:46,583 - psysmysql.stock - INFO - Completado: Generando resumen de stock (-0.006s)
2025-09-04 02:34:46,583 - psysmysql.performance - INFO - psysmysql.services.stock_service.get_stock_summary ejecutado en 0.006s
2025-09-04 02:34:46,584 - psysmysql.stock - INFO - Iniciando: Generando alertas de stock
2025-09-04 02:34:46,587 - psysmysql.stock - INFO - Alertas generadas: 0
2025-09-04 02:34:46,587 - psysmysql.stock - INFO - Completado: Generando alertas de stock (-0.003s)
2025-09-04 02:35:01,245 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor IPS con cantidad 1
2025-09-04 02:35:01,263 - psysmysql.sells - INFO - Nuevo producto Monitor IPS agregado al carrito: 1 unidades
2025-09-04 02:35:01,263 - psysmysql.sells - INFO - Completado: Agregando producto Monitor IPS con cantidad 1 (-0.018s)
2025-09-04 02:35:01,263 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.018s
2025-09-04 02:35:11,532 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 2
2025-09-04 02:35:11,548 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 2 unidades
2025-09-04 02:35:11,548 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 2 (-0.016s)
2025-09-04 02:35:11,548 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.016s
2025-09-04 02:35:21,724 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$580000.0
2025-09-04 02:35:21,738 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$580000.0 (-0.014s)
2025-09-04 02:35:21,738 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 636, 'name': 'Monitor IPS', 'price': 500000.0, 'quantity': 1, 'pricexquantity': 500000.0}, {'id': 637, 'name': 'Bateria', 'price': 40000.0, 'quantity': 2, 'pricexquantity': 80000.0}, {'totals': {'quantity': 3, 'subtotal': 469800.0, 'iva': 110200.0, 'total_sell': 580000.0}}], empleado=francisco, total=$580000.0, tipo_pago=tarjeta credito
2025-09-04 02:35:21,738 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.015s
2025-09-04 02:36:16,631 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 12:31:14,388 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:05:21,786 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:09:06,830 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:39:26,628 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/search_orm.py changed, reloading.
2025-09-04 13:42:07,974 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:42:33,588 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:42:54,726 - psysmysql.sells - INFO - Iniciando: Agregando producto Teclado con cantidad 1
2025-09-04 13:42:54,743 - psysmysql.sells - INFO - Nuevo producto Teclado agregado al carrito: 1 unidades
2025-09-04 13:42:54,743 - psysmysql.sells - INFO - Completado: Agregando producto Teclado con cantidad 1 (-0.017s)
2025-09-04 13:42:54,743 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.017s
2025-09-04 13:43:09,872 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$340000.0
2025-09-04 13:43:09,885 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$340000.0 (-0.012s)
2025-09-04 13:43:09,885 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 638, 'name': 'Teclado', 'price': 340000.0, 'quantity': 1, 'pricexquantity': 340000.0}, {'totals': {'quantity': 1, 'subtotal': 275400.0, 'iva': 64600.0, 'total_sell': 340000.0}}], empleado=francisco, total=$340000.0, tipo_pago=tarjeta credito
2025-09-04 13:43:09,885 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.013s
2025-09-04 13:45:10,257 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:45:36,983 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:45:46,117 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:45:49,771 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:45:52,816 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:46:16,553 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor 4k con cantidad 1
2025-09-04 13:46:16,567 - psysmysql.sells - INFO - Nuevo producto Monitor 4k agregado al carrito: 1 unidades
2025-09-04 13:46:16,567 - psysmysql.sells - INFO - Completado: Agregando producto Monitor 4k con cantidad 1 (-0.014s)
2025-09-04 13:46:16,567 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 13:46:30,456 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$1000000.0
2025-09-04 13:46:30,464 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$1000000.0 (-0.008s)
2025-09-04 13:46:30,464 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 639, 'name': 'Monitor 4k', 'price': 1000000.0, 'quantity': 1, 'pricexquantity': 1000000.0}, {'totals': {'quantity': 1, 'subtotal': 810000.0, 'iva': 190000.0, 'total_sell': 1000000.0}}], empleado=francisco, total=$1000000.0, tipo_pago=Efectivo
2025-09-04 13:46:30,464 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.008s
2025-09-04 13:47:59,108 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:48:08,179 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:48:18,675 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:48:25,919 - psysmysql.sells - INFO - Iniciando: Agregando producto Adaptador usb a/usb c con cantidad 2
2025-09-04 13:48:25,932 - psysmysql.sells - INFO - Nuevo producto Adaptador usb a/usb c agregado al carrito: 2 unidades
2025-09-04 13:48:25,932 - psysmysql.sells - INFO - Completado: Agregando producto Adaptador usb a/usb c con cantidad 2 (-0.013s)
2025-09-04 13:48:25,932 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.013s
2025-09-04 13:48:42,320 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$180000.0
2025-09-04 13:48:42,335 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$180000.0 (-0.014s)
2025-09-04 13:48:42,335 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 640, 'name': 'Adaptador usb a/usb c', 'price': 90000.0, 'quantity': 2, 'pricexquantity': 180000.0}, {'totals': {'quantity': 2, 'subtotal': 145800.0, 'iva': 34200.0, 'total_sell': 180000.0}}], empleado=francisco, total=$180000.0, tipo_pago=transferencia
2025-09-04 13:48:42,335 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.015s
2025-09-04 13:50:18,651 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:50:27,499 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:50:31,191 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:50:42,901 - psysmysql.sells - INFO - Iniciando: Agregando producto Mouse Logitech con cantidad 2
2025-09-04 13:50:42,913 - psysmysql.sells - INFO - Nuevo producto Mouse Logitech agregado al carrito: 2 unidades
2025-09-04 13:50:42,913 - psysmysql.sells - INFO - Completado: Agregando producto Mouse Logitech con cantidad 2 (-0.012s)
2025-09-04 13:50:42,913 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.012s
2025-09-04 13:50:43,360 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:50:56,473 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$400000.0
2025-09-04 13:50:56,485 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$400000.0 (-0.012s)
2025-09-04 13:50:56,485 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 641, 'name': 'Mouse Logitech', 'price': 200000.0, 'quantity': 2, 'pricexquantity': 400000.0}, {'totals': {'quantity': 2, 'subtotal': 324000.0, 'iva': 76000.0, 'total_sell': 400000.0}}], empleado=francisco, total=$400000.0, tipo_pago=Efectivo
2025-09-04 13:50:56,485 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.012s
2025-09-04 13:50:56,963 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 13:52:02,328 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:56:36,399 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 13:57:46,255 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 14:00:55,714 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 14:03:00,919 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 14:03:12,859 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 14:03:19,200 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor IPS con cantidad 1
2025-09-04 14:03:19,216 - psysmysql.sells - INFO - Nuevo producto Monitor IPS agregado al carrito: 1 unidades
2025-09-04 14:03:19,217 - psysmysql.sells - INFO - Completado: Agregando producto Monitor IPS con cantidad 1 (-0.017s)
2025-09-04 14:03:19,217 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.017s
2025-09-04 14:03:25,154 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$500000.0
2025-09-04 14:03:25,165 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$500000.0 (-0.011s)
2025-09-04 14:03:25,165 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 642, 'name': 'Monitor IPS', 'price': 500000.0, 'quantity': 1, 'pricexquantity': 500000.0}, {'totals': {'quantity': 1, 'subtotal': 405000.0, 'iva': 95000.0, 'total_sell': 500000.0}}], empleado=francisco, total=$500000.0, tipo_pago=Efectivo
2025-09-04 14:03:25,165 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.011s
2025-09-04 16:17:59,000 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 16:17:59,034 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 16:17:59,034 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.034s)
2025-09-04 16:17:59,034 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.034s
2025-09-04 16:18:08,237 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor 4k con cantidad 2
2025-09-04 16:18:08,253 - psysmysql.sells - INFO - Nuevo producto Monitor 4k agregado al carrito: 2 unidades
2025-09-04 16:18:08,253 - psysmysql.sells - INFO - Completado: Agregando producto Monitor 4k con cantidad 2 (-0.016s)
2025-09-04 16:18:08,253 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.016s
2025-09-04 16:18:24,275 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$2040000.0
2025-09-04 16:18:24,287 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$2040000.0 (-0.012s)
2025-09-04 16:18:24,287 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 643, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'id': 644, 'name': 'Monitor 4k', 'price': 1000000.0, 'quantity': 2, 'pricexquantity': 2000000.0}, {'totals': {'quantity': 3, 'subtotal': 1652400.0, 'iva': 387600.0, 'total_sell': 2040000.0}}], empleado=francisco, total=$2040000.0, tipo_pago=transferencia
2025-09-04 16:18:24,287 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.012s
2025-09-04 16:27:52,190 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:29:21,117 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 16:37:18,574 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor 4k con cantidad 1
2025-09-04 16:37:18,585 - psysmysql.sells - INFO - Nuevo producto Monitor 4k agregado al carrito: 1 unidades
2025-09-04 16:37:18,586 - psysmysql.sells - INFO - Completado: Agregando producto Monitor 4k con cantidad 1 (-0.011s)
2025-09-04 16:37:18,586 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.012s
2025-09-04 16:37:31,246 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$1000000.0
2025-09-04 16:37:31,259 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$1000000.0 (-0.013s)
2025-09-04 16:37:31,260 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 645, 'name': 'Monitor 4k', 'price': 1000000.0, 'quantity': 1, 'pricexquantity': 1000000.0}, {'totals': {'quantity': 1, 'subtotal': 810000.0, 'iva': 190000.0, 'total_sell': 1000000.0}}], empleado=francisco, total=$1000000.0, tipo_pago=Efectivo
2025-09-04 16:37:31,260 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.014s
2025-09-04 16:37:48,076 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": quantity_email[0]["idproduct"],
              ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'idproduct'
2025-09-04 16:38:22,262 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:38:25,882 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": quantity_email[0]["id_product"],
              ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^
KeyError: 'id_product'
2025-09-04 16:41:41,022 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:41:43,589 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product[0]["idproduct"],
              ~~~~~~~~~~~~~~~^^^^^^^^^^^^^
TypeError: 'SellProducts' object is not subscriptable
2025-09-04 16:49:18,008 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:49:25,907 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product.get("idproduct"),
              ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 16:49:48,131 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:49:50,527 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product.get("idproduct", []),
              ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 619, in get
    clone = self._chain() if self.query.combinator else self.filter(*args, **kwargs)
                                                        ~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1493, in filter
    return self._filter_or_exclude(False, args, kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1511, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1518, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
    ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1646, in add_q
    clause, _ = self._add_q(q_object, can_reuse)
                ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1678, in _add_q
    child_clause, needed_inner = self.build_filter(
                                 ~~~~~~~~~~~~~~~~~^
        child,
        ^^^^^^
    ...<7 lines>...
        update_join_types=update_join_types,
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1523, in build_filter
    arg, value = filter_expr
    ^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
2025-09-04 16:50:25,331 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:50:27,987 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:51:25,242 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:51:35,555 - psysmysql.sells - INFO - Iniciando: Agregando producto Camara con cantidad 1
2025-09-04 16:51:35,568 - psysmysql.sells - INFO - Nuevo producto Camara agregado al carrito: 1 unidades
2025-09-04 16:51:35,568 - psysmysql.sells - INFO - Completado: Agregando producto Camara con cantidad 1 (-0.013s)
2025-09-04 16:51:35,568 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.013s
2025-09-04 16:51:41,064 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$120000.0
2025-09-04 16:51:41,071 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$120000.0 (-0.008s)
2025-09-04 16:51:41,072 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 646, 'name': 'Camara', 'price': 120000.0, 'quantity': 1, 'pricexquantity': 120000.0}, {'totals': {'quantity': 1, 'subtotal': 97200.0, 'iva': 22800.0, 'total_sell': 120000.0}}], empleado=francisco, total=$120000.0, tipo_pago=Efectivo
2025-09-04 16:51:41,072 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.008s
2025-09-04 16:51:53,945 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product.values("idproduct").name,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'name'
2025-09-04 16:52:30,138 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:52:34,252 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2237, in add_fields
    join_info = self.setup_joins(
        name.split(LOOKUP_SEP), opts, alias, allow_many=allow_m2m
    )
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1900, in setup_joins
    path, final_field, targets, rest = self.names_to_path(
                                       ~~~~~~~~~~~~~~~~~~^
        names[:pivot],
        ^^^^^^^^^^^^^^
    ...<2 lines>...
        fail_on_missing=True,
        ^^^^^^^^^^^^^^^^^^^^^
    )
    ^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 1805, in names_to_path
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'idproduct_name' into field. Choices are: idproduct, idproduct_id, idsell, idsell_id, idsell_product, priceunitaty, quantity

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product.values("idproduct_name"),
              ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1368, in values
    clone = self._values(*fields, **expressions)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/query.py", line 1363, in _values
    clone.query.set_values(fields)
    ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2563, in set_values
    self.add_fields(field_names, True)
    ~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/db/models/sql/query.py", line 2265, in add_fields
    raise FieldError(
    ...<2 lines>...
    )
django.core.exceptions.FieldError: Cannot resolve keyword 'idproduct_name' into field. Choices are: idproduct, idproduct_id, idsell, idsell_id, idsell_product, priceunitaty, quantity
2025-09-04 16:53:49,889 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 16:59:45,203 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:00:03,747 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor 4k con cantidad 1
2025-09-04 17:00:03,761 - psysmysql.sells - INFO - Nuevo producto Monitor 4k agregado al carrito: 1 unidades
2025-09-04 17:00:03,761 - psysmysql.sells - INFO - Completado: Agregando producto Monitor 4k con cantidad 1 (-0.013s)
2025-09-04 17:00:03,761 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.014s
2025-09-04 17:00:09,311 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$1000000.0
2025-09-04 17:00:09,321 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$1000000.0 (-0.010s)
2025-09-04 17:00:09,321 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 647, 'name': 'Monitor 4k', 'price': 1000000.0, 'quantity': 1, 'pricexquantity': 1000000.0}, {'totals': {'quantity': 1, 'subtotal': 810000.0, 'iva': 190000.0, 'total_sell': 1000000.0}}], empleado=francisco, total=$1000000.0, tipo_pago=Efectivo
2025-09-04 17:00:09,321 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.010s
2025-09-04 17:00:22,702 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 463, in _handle_sent_form
    "nombre": sell_product.name,
              ^^^^^^^^^^^^^^^^^
AttributeError: 'QuerySet' object has no attribute 'name'
2025-09-04 17:07:09,358 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:09:44,028 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 17:09:59,020 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 17:09:59,031 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 17:09:59,031 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.011s)
2025-09-04 17:09:59,031 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.011s
2025-09-04 17:10:08,424 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$40000.0
2025-09-04 17:10:08,436 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$40000.0 (-0.012s)
2025-09-04 17:10:08,437 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 648, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 1, 'subtotal': 32400.0, 'iva': 7600.0, 'total_sell': 40000.0}}], empleado=francisco, total=$40000.0, tipo_pago=Efectivo
2025-09-04 17:10:08,437 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.013s
2025-09-04 17:11:09,755 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 17:16:02,054 - django.request - WARNING - Not Found: /favicon.ico
2025-09-04 17:16:16,854 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor IPS con cantidad 1
2025-09-04 17:16:16,869 - psysmysql.sells - INFO - Nuevo producto Monitor IPS agregado al carrito: 1 unidades
2025-09-04 17:16:16,869 - psysmysql.sells - INFO - Completado: Agregando producto Monitor IPS con cantidad 1 (-0.015s)
2025-09-04 17:16:16,869 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 17:17:09,279 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor IPS con cantidad 1
2025-09-04 17:17:09,291 - psysmysql.sells - INFO - Nuevo producto Monitor IPS agregado al carrito: 1 unidades
2025-09-04 17:17:09,291 - psysmysql.sells - INFO - Completado: Agregando producto Monitor IPS con cantidad 1 (-0.012s)
2025-09-04 17:17:09,291 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.012s
2025-09-04 17:17:15,619 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 17:17:15,637 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 17:17:15,637 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.019s)
2025-09-04 17:17:15,637 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.019s
2025-09-04 17:17:28,738 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$540000.0
2025-09-04 17:17:28,749 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$540000.0 (-0.011s)
2025-09-04 17:17:28,749 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 650, 'name': 'Monitor IPS', 'price': 500000.0, 'quantity': 1, 'pricexquantity': 500000.0}, {'id': 651, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 2, 'subtotal': 437400.0, 'iva': 102600.0, 'total_sell': 540000.0}}], empleado=francisco, total=$540000.0, tipo_pago=transferencia
2025-09-04 17:17:28,749 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.011s
2025-09-04 17:18:33,461 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 17:26:28,792 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:29:17,598 - psysmysql.sells - INFO - Iniciando: Agregando producto Teclado con cantidad 1
2025-09-04 17:29:17,609 - psysmysql.sells - INFO - Nuevo producto Teclado agregado al carrito: 1 unidades
2025-09-04 17:29:17,610 - psysmysql.sells - INFO - Completado: Agregando producto Teclado con cantidad 1 (-0.012s)
2025-09-04 17:29:17,610 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.012s
2025-09-04 17:29:25,709 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$340000.0
2025-09-04 17:29:25,720 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$340000.0 (-0.011s)
2025-09-04 17:29:25,720 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 652, 'name': 'Teclado', 'price': 340000.0, 'quantity': 1, 'pricexquantity': 340000.0}, {'totals': {'quantity': 1, 'subtotal': 275400.0, 'iva': 64600.0, 'total_sell': 340000.0}}], empleado=francisco, total=$340000.0, tipo_pago=Efectivo
2025-09-04 17:29:25,721 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.012s
2025-09-04 17:29:42,081 - django.request - ERROR - Internal Server Error: /sell-product/
Traceback (most recent call last):
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/contrib/auth/decorators.py", line 59, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/venv/lib64/python3.13/site-packages/django/views/generic/base.py", line 144, in dispatch
    return handler(request, *args, **kwargs)
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 323, in post
    return self._handle_sent_form(request)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^
  File "/home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py", line 516, in _handle_sent_form
    client_email_to_send, email_subject, email_body, email_message
                                                     ^^^^^^^^^^^^^
NameError: name 'email_message' is not defined
2025-09-04 17:39:19,710 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:40:19,714 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:40:46,407 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/views.py changed, reloading.
2025-09-04 17:42:04,088 - psysmysql.sells - INFO - Iniciando: Agregando producto Teclado con cantidad 1
2025-09-04 17:42:04,102 - psysmysql.sells - INFO - Nuevo producto Teclado agregado al carrito: 1 unidades
2025-09-04 17:42:04,102 - psysmysql.sells - INFO - Completado: Agregando producto Teclado con cantidad 1 (-0.015s)
2025-09-04 17:42:04,103 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 17:42:11,653 - psysmysql.sells - INFO - Iniciando: Agregando producto Bateria con cantidad 1
2025-09-04 17:42:11,667 - psysmysql.sells - INFO - Nuevo producto Bateria agregado al carrito: 1 unidades
2025-09-04 17:42:11,668 - psysmysql.sells - INFO - Completado: Agregando producto Bateria con cantidad 1 (-0.014s)
2025-09-04 17:42:11,668 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 17:42:25,910 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$380000.0
2025-09-04 17:42:25,918 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$380000.0 (-0.008s)
2025-09-04 17:42:25,919 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 653, 'name': 'Teclado', 'price': 340000.0, 'quantity': 1, 'pricexquantity': 340000.0}, {'id': 654, 'name': 'Bateria', 'price': 40000.0, 'quantity': 1, 'pricexquantity': 40000.0}, {'totals': {'quantity': 2, 'subtotal': 307800.0, 'iva': 72200.0, 'total_sell': 380000.0}}], empleado=francisco, total=$380000.0, tipo_pago=Efectivo
2025-09-04 17:42:25,919 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.009s
2025-09-04 17:47:59,223 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 17:48:01,996 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
2025-09-04 17:51:35,211 - psysmysql.sells - INFO - Iniciando: Agregando producto Monitor IPS con cantidad 1
2025-09-04 17:51:35,223 - psysmysql.sells - INFO - Nuevo producto Monitor IPS agregado al carrito: 1 unidades
2025-09-04 17:51:35,223 - psysmysql.sells - INFO - Completado: Agregando producto Monitor IPS con cantidad 1 (-0.012s)
2025-09-04 17:51:35,223 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.012s
2025-09-04 17:51:42,324 - psysmysql.sells - INFO - Iniciando: Agregando producto Teclado con cantidad 1
2025-09-04 17:51:42,337 - psysmysql.sells - INFO - Nuevo producto Teclado agregado al carrito: 1 unidades
2025-09-04 17:51:42,338 - psysmysql.sells - INFO - Completado: Agregando producto Teclado con cantidad 1 (-0.014s)
2025-09-04 17:51:42,338 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.014s
2025-09-04 17:51:49,187 - psysmysql.sells - INFO - Iniciando: Agregando producto Adaptador usb a/usb c con cantidad 1
2025-09-04 17:51:49,202 - psysmysql.sells - INFO - Nuevo producto Adaptador usb a/usb c agregado al carrito: 1 unidades
2025-09-04 17:51:49,202 - psysmysql.sells - INFO - Completado: Agregando producto Adaptador usb a/usb c con cantidad 1 (-0.015s)
2025-09-04 17:51:49,202 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.015s
2025-09-04 17:52:31,230 - psysmysql.sells - INFO - Iniciando: Agregando producto Mouse con cantidad 1
2025-09-04 17:52:31,243 - psysmysql.sells - INFO - Nuevo producto Mouse agregado al carrito: 1 unidades
2025-09-04 17:52:31,243 - psysmysql.sells - INFO - Completado: Agregando producto Mouse con cantidad 1 (-0.013s)
2025-09-04 17:52:31,243 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_sell ejecutado en 0.013s
2025-09-04 17:52:51,774 - psysmysql.sells - INFO - Iniciando: Creando registro de venta: total=$1020000.0
2025-09-04 17:52:51,799 - psysmysql.sells - INFO - Completado: Creando registro de venta: total=$1020000.0 (-0.025s)
2025-09-04 17:52:51,800 - psysmysql.sells - INFO - Venta registrada exitosamente: ID=[{'id': 655, 'name': 'Monitor IPS', 'price': 500000.0, 'quantity': 1, 'pricexquantity': 500000.0}, {'id': 656, 'name': 'Teclado', 'price': 340000.0, 'quantity': 1, 'pricexquantity': 340000.0}, {'id': 657, 'name': 'Adaptador usb a/usb c', 'price': 90000.0, 'quantity': 1, 'pricexquantity': 90000.0}, {'id': 658, 'name': 'Mouse', 'price': 90000.0, 'quantity': 1, 'pricexquantity': 90000.0}, {'totals': {'quantity': 4, 'subtotal': 826200.0, 'iva': 193800.0, 'total_sell': 1020000.0}}], empleado=francisco, total=$1020000.0, tipo_pago=transferencia
2025-09-04 17:52:51,800 - psysmysql.sells - INFO - psysmysql.services.sell_service.register_detail ejecutado en 0.026s
2025-09-04 18:12:42,472 - django.utils.autoreload - INFO - /home/Francisco-dev/work/Python/Django/WORK/PsysMsql/psysmysql/services/factura_service.py changed, reloading.
